readme.md

```diff
@@ -49,4 +49,19 @@ RUNTIME_MODE    | no       | production             | Running mode (development
 
 ### Other Configuration Options
 - Environment variables can also be set in ./config.json for development. If both provide a value for a specific property, the environment variable will override the file. You can define your own config json file and pass it to the application with `-c <config_file>`
-- The application can take any flags for [glog](https://github.com/golang/glog), it will pass them straight into glog. The glog flag `--logtostderr` is set to true by default.
\ No newline at end of file
+- The application can take any flags for [glog](https://github.com/golang/glog), it will pass them straight into glog. The glog flag `--logtostderr` is set to true by default.
+
+
+### Contribution
+When making contributions to the search collector project, fork the project then merge your chages by creating a pull request from your personal fork to the main project.
```

Comment:
1. Keep generic to make it easy to copy/paste and maintain across our repos.
2. Fix typo.

```suggestion
When making contributions to this project, fork the project then merge your changes by creating a pull request from your personal fork to the main project.
```

Reply:
I suggest that the statement reads: 

When you make contributions to this project, fork the project  then merge your changes by creating a pull request from your personal fork to the main project:

Reply:
👍